test/JDBC/upgrade/14_9/schedule

```diff
@@ -0,0 +1,411 @@
+# Schedule File for JDBC Test Framework for local run
```

Comment:
Is it the copy of current latest/schedule file?

Reply:
Yes